azure-pipelines-integration-corehost.yml

```diff
@@ -12,11 +12,17 @@ trigger:
 # Automatic building of all PRs is disabled in the pipeline's trigger page.
 # See https://docs.microsoft.com/en-us/azure/devops/pipelines/repos/github?view=azure-devops&tabs=yaml#comment-triggers
 pr:
-- main
-- main-vs-deps
-- release/*
-- features/*
-- demos/*
+  branches:
```

Comment:
I don't see a need to try it immediately but: there's no way to template this section out, correct?

Reply:
No idea if that's possible tbh.